nanodex/cli/shell.py

```
@@ -0,0 +1,86 @@
+
+    # Initialize persistent context
+    ctx.ensure_object(dict)
+    ctx.obj["shell_mode"] = True
+
+    # Load config once (instead of per-command)
+    try:
+        from ..utils.config import Config
+
+        ctx.obj["config"] = Config("config.yaml")
+        console.print("[green]✓[/green] Configuration loaded\n")
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)